### PR TITLE
Add usrsctp project

### PR DESCRIPTION
**Pull request by weinrank**
Hi,

we would like to integrate our existing fuzzer suite into the OSS-FUZZ system.

### CODE REVIEW

**Comment by weinrank** [comment]
@googlebot I signed it!

**Comment by inferno-chromium** [comment]
Can you comment on the planned use of usrsctp ? Is it part of any distros yet ?

**Comment by weinrank** [comment]
For example: SCTP the underlying transport protocol for WebRTC Data-Channels and the usrsctp library is used within Chrome and Firfefox.

**Comment by weinrank** [comment]
References

https://chromium.googlesource.com/chromium/src/+/master/third_party/usrsctp/

https://dxr.mozilla.org/mozilla-central/source/netwerk/sctp/src/usrsctp.h

File: projects/usrsctp/project.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/sctplab/usrsctp"
+primary_contact: "weinrank@fh-muenster.de"
+auto_ccs:
+  - "tuexen@fh-muenster.de"
+sanitizers:
+  - address
+  - memory:
+    experimental: True
```
Comment (inferno-chromium) [inline projects/usrsctp/project.yaml]: Please remove the experimental flag here. This should not be needed unless your library has system dependencies and which is not in our instrumented list.

Comment (weinrank) [reply projects/usrsctp/project.yaml]: Done!

**Comment by inferno-chromium** [comment]
Please fix the travis failures, once it passes, then we can merge

```
/src/build.sh: line 20: cd: fuzzer: No such file or directory
Fuzzers build failed.
Failed projects: usrsctp
The command "./infra/travis/travis_build.py" exited with 1.
```

**Comment by weinrank** [comment]
The fuzzer hasn't been merged into usrsctp's master branch.
Once the PR has been merged, I'll trigger the travis build again.
https://github.com/sctplab/usrsctp/pull/356

**Comment by weinrank** [comment]
We've integrated the fuzzer into the repo.
Travis seems to be happy now.

**Pull request merged**